one/src/lib.rs

```diff
@@ -47,9 +43,9 @@ struct Cli {
 enum Command {
     /// Run a daemon process
     Daemon(DaemonOpts),
-    /// Event store tools
+    /// Perform various migrations
     #[command(subcommand)]
-    Events(events::EventsCommand),
+    Migrations(migrations::EventsCommand),
```

Comment:
is this going to get confusing with the database migrations that @dav1do is working on right now?

Reply:
No, I expect he will plug into this top level command. The intent is that operators can expect to find all types of migration commands in one place regardless of if they are within the same db or across dbs.